### PR TITLE
Update deps to fix React Reconciler error

### PR DESCRIPTION
**Pull request by jthegedus**
Was getting this error in with-glamorous example. Thought I would update deps in both.

with-glamorous-errors

### CODE REVIEW

File: examples/with-glamor/package.json [hunk 1]
```diff
@@ -7,10 +7,10 @@
     "start": "next start"
   },
   "dependencies": {
-    "glamor": "^2.20.24",
-    "next": "latest",
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0"
+    "glamor": "2.20.40",
+    "next": "4.2.1",
```
Comment (timneutkens) [inline examples/with-glamor/package.json]: Lets keep this `latest`

File: examples/with-glamorous/package.json [hunk 2]
```diff
@@ -7,11 +7,11 @@
...
   },
   "dependencies": {
-    "glamor": "^2.20.24",
-    "glamorous": "^4.11.0",
-    "next": "^2.0.1",
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0"
+    "glamor": "2.20.40",
+    "glamorous": "4.11.2",
+    "next": "4.2.1",
```
Comment (timneutkens) [inline examples/with-glamorous/package.json]: Same here, lets make it `latest`

**Comment by jthegedus** [comment]
@timneutkens I did remember seeing a discussion around this, but couldn't see it listed anywhere in the docs. Maybe it would be worth creating a README or Contribution guideline for the examples. It **might** reduce these small issues.

**Review by timneutkens — APPROVED** [review]

**Comment by timneutkens** [comment]
Fixed in canary.

**Pull request closed without merging**